packages/block-editor/src/components/colors/test/utils.js

```diff
@@ -108,11 +108,5 @@ describe( 'color utils', () => {
 				getColorClassName( 'background', undefined )
 			).toBeUndefined();
 		} );
-
-		it( 'should return a class name with the color slug in kebab case', () => {
-			expect( getColorClassName( 'background', 'Light Purple' ) ).toBe(
```

Comment:
So what happens with cases like this with `spaces`? Wouldn't they break this by creating more css classes?

Reply:
https://github.com/WordPress/gutenberg/issues/32347#issuecomment-851579181 :) I think we should make sure the classes don't have spaces. There are a few options here, I'm inclined to converting them to `-`.

Reply:
Fixed at https://github.com/WordPress/gutenberg/pull/32352/commits/def68c83fecdf6d3190b66098c5a4b488908cc87 including folding multiple consecutive spaces into a single `-`.